js/forum_italia.js

```diff
@@ -0,0 +1,295 @@
+        offset: '175px, 40px',
+        trigger: 'focus'
+      });
+
+      // Logout modal
+      $('body').append(that.logoutModal);
+      $('#logout-modal__submit').click(forumItalia.logout);
+
+      if (that.client.isCurrentUserSilenced()) {
+        return forumItalia.setSilencedMarkup();
+      }
+
+      // Minimum characters
+      $('textarea.box-comment__body').bind('input', function() {
+        $requiredChars = $(this).parents('div.box-comment').find('.required-chars');
+        $requiredMessage = $(this).parents('div.box-comment').find('.box-comment__required');
+        $sendButton = $(this).parents('div.box-comment').find('.box-comment__submit');
+
+        var nedeedChars = 20 - $(this).val().replace(/ /g,'').length;
+        if (nedeedChars > 0) {
+          $requiredMessage.show();
+          $requiredChars.text(nedeedChars);
+          $sendButton.attr('disabled', true);
+        } else {
+          $requiredMessage.hide();
+          $sendButton.attr('disabled', false);
+        }
+      });
+
+      // Send message button
+      $('button.box-comment__submit').click(function() {
+        var $boxComment = $(this).parents('.box-comment');
+        var $messageTextArea = $boxComment.find('.box-comment__body')
+        var $sendButton = $boxComment.find('.box-comment__submit');
+        var messageBody = $messageTextArea.val().trim();
+        var $errors = $boxComment.find('.box-comment__errors-box');
+        var topicId = $boxComment.data('topic');
+        var $commentList = $boxComment.parents('.block-comments__body').find('.block-comments__list');
+
+        if (messageBody === '') {
+          return false;
+        } else {
+          $boxComment.addClass('sending');
+          // Disable textarea & submit button
+          $messageTextArea.attr('disabled', true);
+          $sendButton.attr('disabled', true);
+          that.client.postMessage(topicId, messageBody)
+            .then(function(response) {
+              $messageTextArea.val('');
+              $errors.removeClass('text-danger').html('');
+              forumItalia.refreshCommentsMarkup(topicId, $commentList, response.id);
+            })
+            .catch(function(error) {
+              $errors.addClass('text-danger').html(error.join('<br>'));
+            }).finally(function() {
+              $boxComment.removeClass('sending');
+              // Enable textarea & submit button
+              $messageTextArea.attr('disabled', false);
+              $sendButton.attr('disabled', false);
+            });
+        }
+      });
+
+      forumItalia.refrestAllCommentsMarkup();
+    },
+
+    setLoggedOutMarkup: function() {
+      $(that.commentBoxesSelector).each(function() {
+        $(this).html(that.forumLoginMarkup).find('.login-button').click(forumItalia.login);
+      });
+
+      forumItalia.refrestAllCommentsMarkup();
+    },
+    
+    setSilencedMarkup: function() {
+      $('textarea.box-comment__body').attr('disabled', true);
```

Comment:
Se hai verificato che tutto sia a posto ok, altrimenti vale la pena [fare una prova](https://stackoverflow.com/questions/13626517/how-to-remove-disabled-attribute-using-jquery/13626565#13626565) utilizzando `.prop()`.